docs/src/modules/components/branding/Logo.js

```diff
@@ -0,0 +1,27 @@
+import * as React from 'react';
+import PropTypes from 'prop-types';
+import { experimentalStyled as styled } from '@material-ui/core/styles';
+import Box from '@material-ui/core/Box';
+import t1 from 'docs/src/modules/components/branding/t1';
+
+const LogoRoot = styled(Box)({
+  height: 32,
+});
+
+export default function Logo(props) {
+  const { href, ...other } = props;
+  return (
+    <Box component={href ? 'a' : 'div'} href={href}>
+      <LogoRoot
+        {...other}
+        component="img"
+        src="/static/images/branding/full-logo.svg"
+        alt={t1('Material-UI Logo')}
+      />
+    </Box>
+  );
+}
+
+Logo.propTypes = {
+  href: PropTypes.string,
+};
```

Comment:
What do you think about writing these new pages in TypeScript?

```suggestion
```

Reply:
Makes sense to move the new pages to `Typescript`. I'll work on this tomorrow 👍